docs/source/contributor-guide/index.md

```diff
@@ -62,72 +63,6 @@ unable to make progress you should unassign the issue by using the `unassign me`
 link at the top of the issue page (and ask for help if are stuck) so that
 someone else can get involved in the work.
 
-### Discussing New Features
```

Comment:
I just moved this content out from a sub bullet and into a more discoverable place on Roadmap